lib/gooddata/lcm/actions/collect_data_product.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require_relative 'base_action'
+
+module GoodData
+  module LCM2
+    class CollectDataProduct < BaseAction
```

Comment:
Please split this action into `EnsureDataproduct` and `CollectDataproduct`. Use the `EnsureDataproduct` only in `ReleaseBrick`.